packages/core/src/payment/strategies/paypal-commerce/paypal-commerce-request-sender.ts

```diff
@@ -27,6 +28,9 @@ export default class PaypalCommerceRequestSender {
         } else if (isCredit) {
             provider = 'paypalcommercecredit';
         }
+        if (isVenmo && !isAPM) {
+            provider = isCheckout ? 'paypalcommercevenmocheckout' : 'paypalcommercevenmo';
```

Comment:
Can you help me to understand why we use `paypalcommercevenmocheckout` provider name for PPC Venmo button strategy instead of `paypalcommercevenmo`?

Reply:
We need to send the name of the provider to the backend depending on where the request was sent from the checkout page or not. The backend configures the display of the button title in the PayPal popup. If the request was sent from the checkout page then the name is Pay Now, if not then Continue